### ISSUE TITLE
No `Asset.recording_times` prop in the `Asset` type

### ISSUE THREAD

**Issue by branks** [muxinc/mux-node-sdk#188]
According to https://docs.mux.com/api-reference/video#tag/assets - there is a prop called `recording_times` but it does't exist on the type for Asset - https://github.com/muxinc/mux-node-sdk/blob/6e479cc38ed7083629f97d5991ab4359e4c491a6/src/video/domain.ts#L126

**Comment by jaredsmith** [comment]
I'm looking into this now.

**Comment by branks** [comment]
Thanks, I built a temporary type in my code in the meantime if it's of any help

```

type RecordingTime = {
  type: string;
  started_at: string;
  duration: number;
}

type MuxAssetWithRecordingTimes = {
  recording_times: RecordingTime[];
}
```

**Comment by jaredsmith** [comment]
This should now be fixed in the 6.3.0 release. Can you please test it out and let me know if that solves your problem?

**Comment by branks** [comment]
Looks like it, thanks!
